src/get-started/create-service.md

```diff
@@ -53,25 +55,34 @@ In our case, this is where we are going to place the service code.
 Let’s start from importing crates with necessary types:
 
 ```rust
```

Comment:
It may make sense to use the [collapsible code sample](https://squidfunk.github.io/mkdocs-material/extensions/pymdown/#details) here:

```markdown
??? note "Imports"
    ```rust
    // ...
    ```
```